asm/berry_pots_app_rodata.s

```diff
@@ -0,0 +1,313 @@
+	.include "asm/macros.inc"
+	.include "overlay_17.inc"
+	.include "global.inc"
+
+	.rodata
+
+; FIXME: Convert all of these to C structs/arrays and figure out what order they go in.
```

Comment:
Were you running into issues with this or just don't want to do it for this PR?

Reply:
They have to be in a certain order which I could not figure out. After doing some trial & error for a couple hours I just gave up